packages/@okta/vuepress-site/docs/guides/password-import-hook/web-server/nodejs/web-server.md

```diff
@@ -0,0 +1,5 @@
+```javascript
+const app = express();
+const port = process.env.PORT || 3000 ;
```

Comment:
Should we put a note somewhere that this example uses HTTP but Okta will require HTTPS so such a server should be placed behind a secure reverse proxy?

Reply:
As discussed, that's covered in password-import-hook/web-server/index.md: 
```
Inline Hook requests sent by Okta are HTTPS requests. To listen for them, your external service needs to implement an HTTPS server provisioned with a certificate from a trusted Certificate Authority.

This guide does not cover the specifics of this part of implementing an external service. In the following example code, we just listen for HTTP requests and assume that our code is running behind a reverse proxy configured for HTTPS.

>**Note:** Okta will not call a non-HTTPS external service endpoint. You must implement HTTPS in order to use Inline Hooks. 
```